src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -124,71 +119,108 @@ int FilterParserThread::parseDATFilterFile()
         return ruleCount;
     }
 
-    const QByteArray data = file.readAll();
+    static const int bufferSize = 2 * 1024 * 1024; // 2 MiB
```

Comment:
After I do the .p2p refactoring I'll transfer this inside the anonymous namespace so it can be accessed by both parsing functions.